commands/run.js

```diff
@@ -56,7 +44,11 @@ module.exports.run = (args, opts) => {
     }
 
     return invoke(team.key, service, action, data, true).then((response) => {
-      console.log(response.body);
+      if (Buffer.isBuffer(response.file)) {
+        process.stdout.write(response.file);
```

Comment:
Same as above. Maybe this piece can be abstracted out, might make it easier to test.